xlive/H2MOD.cpp

```diff
@@ -293,7 +292,7 @@ signed int __cdecl object_new_hook(void *pObject)
 	object_to_variant[result] = variant_index;
 
 	//wchar_t DebugText[255] = { 0 };
-	//ZeroMemory(DebugText, sizeof(DebugText));
```

Comment:
please revert back to ZeroMemory unless you have a real reason to secure this data